src/text_statistics/TextStatistics.java

```diff
@@ -0,0 +1,66 @@
+package text_statistics;
+
+public class TextStatistics {
```

Comment:
To jest tak zwana klasa `Utlis` powinna mieć ona jeszcze prywatny konstruktor i być finalna. Wspomnę dzisiaj o tym na zajęciach.